From c3f041b02819898ba29fb21b46f08836042ada56 Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Mon, 5 Dec 2022 08:57:06 +0800
Subject: [PATCH] Revert "Linux 5.10.157"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit f4245f05389c29c0d556fea359b2fcfd8dce7bdb.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index bf22df29c4d8..166f87bdc190 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 157
+SUBLEVEL = 156
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.3

